pkg/build/build.go

```diff
@@ -1,56 +1,32 @@
 package build
 
 import (
-	_ "embed"
 	"fmt"
-	"os"
 	"path/filepath"
-	"slices"
 
 	"github.com/suse-edge/edge-image-builder/pkg/config"
+	"github.com/suse-edge/edge-image-builder/pkg/context"
 )
 
-//go:embed scripts/script_base.sh
-var combustionScriptBaseCode string
+type configureCombustion func(ctx *context.Context) error
```

Comment:
Using such definition offers the following benefits:
- `combustion` package is not a dependency of the `build` package
- `configureCombustion` can easily be mocked